src/main/java/svc/models/PhoneNumber.java

```diff
@@ -0,0 +1,5 @@
+package svc.models;
+
+public class PhoneNumber {
```

Comment:
You really don't need an entire class for one attribute. That's over engineering at its finest. hehe. Just return the String from the manager.